content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst

```diff
@@ -4,34 +4,118 @@ Inventory adjustments
 
 In any warehouse management system, the recorded inventory counts in the database might not always
 match the actual inventory counts in the warehouse. The discrepancy between the two counts can be
-due to theft, damages, human errors, or other factors. As such, inventory adjustments must be made
+due to damages, human errors, theft, or other factors. As such, inventory adjustments must be made
 to reconcile the differences, and ensure that the recorded counts in the database match the actual
 counts in the warehouse.
 
-In Odoo, inventory adjustments can be accessed through :menuselection:`Inventory --> Operations -->
-Inventory Adjustments`.
+Inventory adjustments page
+==========================
 
-The :guilabel:`Inventory Adjustments` page shows all products that are currently in stock. Each
-line contains the following information:
+To view the :guilabel:`Inventory Adjustments` page, navigate to :menuselection:`Inventory app -->
+Operations --> Inventory Adjustments`.
 
-- :guilabel:`Product`
-- :guilabel:`Lot/Serial Number`
-- :guilabel:`On Hand Quantity`
+On the :guilabel:`Inventory Adjustments` page, all products that are currently in stock are listed,
+with each product line containing the following information:
+
+.. image:: count_products/count-products-inventory-adjustments-page.png
+   :align: center
+   :alt: Inventory adjustments page.
+
+- :guilabel:`Location`: the specific location in the warheouse where a product is stored.
+- :guilabel:`Product`: the product whose quantity is listed on the inventory adjustment line.
+- :guilabel:`Lot/Serial Number`: the specific tracking identifier assigned to the specific product
+  listed. It can contain letters, numbers, or a combination of both.
+
+.. note::
+   If a specific product has a quantity of more than `1.00` in stock, and more than one serial
+   number (or lot number) assigned to it, each uniquely-identified product is displayed on its own
+   product line with its own lot/serial number, displayed under the :guilabel:`Lot/Serial Number`
+   column.
+
+- :guilabel:`On Hand Quantity`: the quantity of the product currently recorded in the database.
+- :guilabel:`UoM`: the *unit of measure* in which the product is measured. Unless otherwise
+  specified (i.e., in **Pounds** or **Ounces**), the default :abbr:`UoM (unit of measure)` is
+  **Units**.
```

Comment:
These are field values no? So a :guilabel: would be appropriate rather than bold 🙂
```suggestion
  specified (i.e., in :guilabel:`Pounds` or :guilabel:`Ounces`), the default :abbr:`UoM (unit of
  measure)` is :guilabel:`Units`.
```